packages/smooth_app/lib/pages/onboarding/consent_analytics_page.dart

```diff
@@ -115,60 +77,56 @@ class ConsentAnalytics extends StatelessWidget {
     );
   }
 
+  BottomAppBar _buildBottomAppBar(
+      BuildContext context, AppLocalizations appLocalizations) {
+    return BottomAppBar(
+      child: ButtonBar(
+        alignment: MainAxisAlignment.spaceBetween,
+        mainAxisSize: MainAxisSize.max,
+        children: <Widget>[
+          _buildButton(
+              context,
+              appLocalizations.refuse_button_label,
+              const Icon(
+                Icons.close_rounded,
+              ),
+              false),
+          _buildButton(
+              context,
+              appLocalizations.authorize_button_label,
+              const Icon(
+                Icons.check_rounded,
+              ),
+              true),
```

Comment:
Just for syntax, could you add an extra comma please?

Reply:
✅